packages/console/src/components/Table/index.tsx

```diff
@@ -33,12 +36,15 @@ const Table = <
   rowGroups,
   columns,
   rowIndexKey,
-  onClickRow,
+  clickRowHandler,
   className,
   headerClassName,
   bodyClassName,
   isLoading,
+  placeholderTitle,
+  placeholderContent,
```

Comment:
titlePlaceholder, contentPlaceHolder. Maybe defining a PlaceHolder type is better.  {title: string, content: ReactNode, Image: ReactNode}

Reply:
cool updated.